swift/apple/FirezoneKit/Sources/FirezoneKit/Views/FirstTimeView.swift

```diff
@@ -23,7 +21,7 @@ import SwiftUI
             .padding(.horizontal, 10)
           Spacer()
           Text(
-            "You can sign in by clicking the Firezone icon in the macOS menu bar or clicking 'Open menu' below."
+            "You can sign in by clicking the Firezone icon in the macOS menu bar."
```

Comment:
Is it possible that having too many icons in the macOS menu bar might make it impossible to open the Firezone menu?

See the PR introducing this feature: https://github.com/firezone/firezone/pull/5505